docs/source-lit/api_reference/api/lightning_app.core.LightningApp.rst

```diff
@@ -0,0 +1,10 @@
+﻿.. role:: hidden
```

Comment:
I think these files should probably be sym-linked? @Borda to confirm?

Reply:
fixed :)